### ISSUE TITLE
Real audio with -declaim

### ISSUE THREAD

**Issue by dave-irvine** [lhartikk/ArnoldC#91]
For almost all of the keywords you have links to YouTube clips of the real audio. While it would certainly take longer during compilation, would it not be possible to fetch the audio from those clips when you specify `-declaim`. Perhaps an extra option to enable this longer compilation.

**Comment by moul** [comment]
:+1:

**Comment by picaron** [comment]
yeah I was disappointed that the declaim was not generated in Arnold's voice

definitely a must
